packages/redux-subspace-observable/src/index.js

```diff
@@ -11,4 +11,7 @@ import { createEpicMiddleware as baseCreateEpicMiddleware } from 'redux-observab
 
 export { default as subspaced } from './observable/subspaced'
 
-export const createEpicMiddleware = (rootEpic, options) => applyToRoot(baseCreateEpicMiddleware(rootEpic, options))
+export const createEpicMiddleware = (rootEpic, options = {}) =>
```

Comment:
@evertbouw @jayphelps, is it safe to assume dependencies is an object, or should we need to cater for cases where something else (array, primitive, function, etc.) are passed?

Reply:
hehe I thought about the same thing but didn't mention it because, while nothing in redux-observable enforces it being an object, not doing so would be weird and discouraged.

Reply:
not saying you can't be more defensive and assert that it is 😄